pages/payruns/[guid]/index.js

```diff
@@ -67,6 +70,13 @@ const SinglePayRun = () => {
 
   return (
     <Container>
+      <HoldPayment
```

Comment:
move this component to inside PayRunItem, pass update through to PayRunItem as a prop and move the [invoidId,setInvoiceId] line 38 to inside payRunItem